RIS.js

```diff
@@ -1453,18 +1453,14 @@ function applyValue(item, zField, value, rawLine) {
 			}
 		break;
 		case 'DOI':
-			if (!ZU.fieldIsValidForType("DOI", item.itemType) && ZU.cleanDOI(value)) {
-				value = ZU.cleanDOI(value);
+			value = ZU.cleanDOI(value);
+			if (!ZU.fieldIsValidForType("DOI", item.itemType) && value) {
 				if(item.extra) {
 					item.extra += '\nDOI: ' + value;
 				} else {
 					item.extra = 'DOI: ' + value;
 				}
 			}
-			else {
-				item[zField] = value;
-			}
-		break;
```

Comment:
This change I cased with my previous comment, might produce now that the doi data will be saved in `item.extra` as well as in `item.DOI` in the translator obejct as you outlined in https://github.com/zotero/translators/pull/1233#discussion_r97809580 . Thus, we might switch back here again to the else case and the break to prevent that.